### DISCUSSION TITLE
R5.0.0 Sunbird ED Regression Open Defects Status

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#150] [General Discussion]
Hi All,

    Please find the open defects and share the ETA for the defect fix as today we will need to close all the defects and provide the sign off by tomorrow 13th Sep 2022,

**@rhwarrier @vinukumar Please find the open defects for Knwlg BB**

![image](https://user-images.githubusercontent.com/40622169/189601007-cd1fdf66-b2a6-4a12-8df9-36d073a7c2a6.png)

**@rhwarrier @reshmi-nair Please find the Open Defects for Lern BB**

![image](https://user-images.githubusercontent.com/40622169/189601360-29068779-4e64-44d9-8486-25bec9026c19.png)

**@luckynvdu @sharathkashyap @swayangjit Please find the Open Defects for Sunbird ED**

![image](https://user-images.githubusercontent.com/40622169/189602248-b42fdc5c-b758-4645-91ca-77e4709d016f.png)

**@tarang27 @surendrasinghs  Please find the Open Defects for Samagra - Prashnavali** 

![image](https://user-images.githubusercontent.com/40622169/189602613-1624bb16-0ce7-4c0c-9c83-573d88582a0a.png)

**[@pallakartheekreddy](https://github.com/pallakartheekreddy) @surendrasinghs Please find the Open Defects for Sunbird Cokreat**

![image](https://user-images.githubusercontent.com/40622169/189602840-663d0a27-a557-4c38-8f3d-c2af328a0cb3.png)

**@vijiurs @aks30 Please find the Open Defects for Sikshalokam**

![image](https://user-images.githubusercontent.com/40622169/189603360-ac9d38da-0ec6-46ff-a61e-bebdca912ee2.png)

**@Prerna789 @surendrasinghs Please find the Open Defects for Samagra UCI**

![image](https://user-images.githubusercontent.com/40622169/189603756-988dfd8f-dd39-4afb-8ada-941ff6bef2bc.png)

CC @chitranshu-keshav @ashokreddy1208  @anandvarada @ShailandraOjha

**Comment by ShailandraOjha** [comment]
Hi @rhwarrier @vinukumar @reshmi-nair @luckynvdu @sharathkashyap @swayangjit @tarang27 @surendrasinghs [@pallakartheekreddy](https://github.com/pallakartheekreddy) @vijiurs @aks30 @Prerna789

Please help us with the fixes on the above issues by today's EOD so that we can verify the issues and execute the failed and blocked test cases by tomorrow to sign off. Else it will be difficult for us to sign off R5.0 tomorrow.

cc: @chitranshu-keshav @ashokreddy1208 @anandvarada
